packages/juce-blueprint/package.json

```diff
@@ -3,27 +3,22 @@
   "version": "0.1.0",
   "description": "BlueprintPlugin user interface in React.",
   "main": "./cjs/Blueprint.js",
+  "types": "./cjs/Blueprint.d.ts",
   "scripts": {
-    "build": "babel esm --out-dir cjs",
+    "build": "tsc",
     "init": "node scripts/init.js"
   },
   "author": "Nick Thompson",
-  "babel": {
-    "presets": [
-      "@babel/preset-env",
-      "@babel/preset-react"
-    ]
-  },
   "devDependencies": {
-    "@babel/cli": "^7.2.0",
-    "@babel/core": "^7.2.0",
-    "@babel/preset-env": "^7.2.0",
-    "@babel/preset-react": "^7.0.0",
+    "@types/invariant": "^2.2.33",
+    "@types/react": "^16.9.41",
     "chalk": "^2.4.2",
-    "fs-extra": "^8.1.0"
+    "fs-extra": "^8.1.0",
+    "typescript": "^3.9.5"
   },
   "dependencies": {
     "core-js": "2.6.0",
+    "invariant": "^2.2.4",
```

Comment:
@nick-thompson 
this lib is used in the `BlueprintRenderer.js` https://github.com/nick-thompson/blueprint/blob/af0e1390c6cb0c0d05ba96abc8ba4a2031b60f95/packages/juce-blueprint/esm/lib/BlueprintRenderer.js#L5 and I'm actually sure where it was coming from originally.

Reply:
Ah right, I think that comes in in the react package, but I think it's good to have it here explicitly